### PR TITLE
fix(os.tmpdir()) strip trailing slash on *unix machines

### PR DESCRIPTION
**Pull request by cirospaciari**

### CODE REVIEW

File: src/bun.js/node/node_os.zig [hunk 1]
```diff
@@ -656,7 +633,11 @@ pub const Os = struct {
                     break :brk systemdir_ ++ "\\temp";
                 }
             } else {
-                break :brk bun.asByteSlice(bun.getenvZ("TMPDIR") orelse bun.getenvZ("TMP") orelse bun.getenvZ("TEMP") orelse "/tmp");
```
Comment (Jarred-Sumner) [inline src/bun.js/node/node_os.zig:659]: instead of this we can do

File: src/bun.js/node/node_os.zig [hunk 2]
```diff
@@ -656,7 +633,11 @@ pub const Os = struct {
...
+                var dir = bun.asByteSlice(bun.getenvZ("TMPDIR") orelse bun.getenvZ("TMP") orelse bun.getenvZ("TEMP") orelse "/tmp");
+                if (dir.len > 1 and strings.endsWithChar(dir, '/')) {
+                    dir = dir[0 .. dir.len - 1];
+                }
+                break :brk dir;
```
Comment (Jarred-Sumner) [inline src/bun.js/node/node_os.zig]: `strings.withoutTrailingSlash(dir)`

Comment (cirospaciari) [reply src/bun.js/node/node_os.zig]: done https://github.com/oven-sh/bun/pull/2276/commits/256281b61024d97255190f5ce9a4438a905a96b5

**Comment by Jarred-Sumner** [comment]
Thanks

**Pull request merged**